### PR TITLE
Add multi-package brief doc

### PR DESCRIPTION
**Pull request by samuel-williams-da**
Add standalone documentation for daml multi-package for internal use.

### CODE REVIEW

**Review by dylant-da — APPROVED** [review]
Best if we get approval from someone who wasn't an author - I've forwarded to Curtis.

**Comment by samuel-williams-da** [comment]
Maybe someone from the docs team?

**Comment by dylant-da** [comment]
From Curtis:
> I think it could benefit from a summary up front:  what it is for, common command line, explanation of a new yaml file and why.  Probably 2-3 paragraphs.

File: unreleased/multi-package.md
Comment (hrischuk-da) [inline unreleased/multi-package.md:229]: Would it make sense that all of the directory trees have the multi-package.yaml at the top of the list, below the .git directory?  This is easier to notice.

Comment (dylant-da) [reply unreleased/multi-package.md]: fixed

File: unreleased/multi-package.md
Comment (marciw-da) [inline unreleased/multi-package.md:49]:
```suggestion
- The developer must always remember to build in order to propagate changes to `package-Logic-1.0.0.dar`.
- If the developer forgets, compilation may succeed without warnings due to the stale `package-Model-1.0.0.dar`, where it would fail if `package-Model-1.0.0.dar` were up to date.
```

File: unreleased/multi-package.md
Comment (marciw-da) [inline unreleased/multi-package.md:392]:
```suggestion
The `--output` flag can be used to specify the location of the DAR generated by `daml build`. This is not new, however it is important to note that the multi-package feature extends the use cases of this flag.
```

Comment (marciw-da) [reply unreleased/multi-package.md:392]: I didn't mark all hyphenation issues or punctuation issues generally -- will do that in the Daml docs repo

File: unreleased/multi-package.md
Comment (marciw-da) [inline unreleased/multi-package.md:1]: I would think of this as a "command" rather than an entire "CLI," but I also think a more general title like "Multi-Package Builds" would be even better.

File: unreleased/multi-package.md
Comment (marciw-da) [inline unreleased/multi-package.md:11]:
```suggestion
### Overview
```

File: unreleased/multi-package.md
Comment (marciw-da) [inline unreleased/multi-package.md:67]: In the Daml docs, we can think about ways to thread the example throughout while also making it more concise -- users might not need all the details, especially later in the doc

**Review by marciw-da — APPROVED** [review]
Nice doc! 👍 I made some high-level comments and minor suggestions. Approving because the changes are optional for this doc and we'll make them in the Daml docs repo either way. 

I decided not to edit for Daml docs style yet, to avoid duplicating effort and also unblock you for now. I'll get in touch separately about where this might live in the Daml docs, as well as some style tips if you do want to make those changes now.

**Comment by marciw-da** [comment]
> From Curtis:
> 
> > I think it could benefit from a summary up front:  what it is for, common command line, explanation of a new yaml file and why.  Probably 2-3 paragraphs.

Agreed, and I can help with the intro when we move this to the Daml docs repo

File: unreleased/multi-package.md
Comment (samuel-williams-da) [reply unreleased/multi-package.md:462]: Alright, lets discuss this once a PR to the docs repo is open, as I'm not familiar with other related pages.

File: unreleased/multi-package.md
Comment (samuel-williams-da) [reply unreleased/multi-package.md:215]: I've updated to developers here, in case we do keep this version

File: unreleased/multi-package.md
Comment (samuel-williams-da) [reply unreleased/multi-package.md:3]: I've rewritten it somewhat, let me know what you think

File: unreleased/multi-package.md
Comment (samuel-williams-da) [reply unreleased/multi-package.md:67]: Right okay, i would very much appreciate the help on that

**Review by marciw-da — APPROVED** [review]
Sorry, accidentally closed this (meant to close the one in the docs repo).

Approving as-is since this is internal and we have an external version now

**Pull request merged**
